### PR TITLE
feat(code): add vscode-arm

### PR DESCRIPTION
**Pull request by fsx950223**
Fixes #

(let GitHub automatically close an issue when this pull request gets merged)

## Before you submit a pull request
This template is not neccessary when you do simple things like updating packages to the latest version. But in doubt, it's always better so provide some information.

## Description
Provide a description, what your changes do and why they are important
add vscode arm version
Please link issues and other pull requests connected to this one.

## Addtional information
Mention things we might need to know. Like:

Works properly:
- [ ] x86_64
- [ ] aarch64 (reasons why it doesn't)

---

## That's it
Thank you for submitting your pull request.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.

### CODE REVIEW

**Comment by JL2210** [comment]
Thanks! Has this been tested?

**Comment by JL2210** [comment]
You want to use the same version for all of them. `1.28.0` doesn't work on Chrome OS yet. Also, change ```
case ARCH
    when 'x86_64', 'i686', 'armv7l'
```
to
```
case ARCH
    when 'x86_64', 'i686', 'armv7l', 'aarch64'
```
Other than that, I'd say this is OK.

**Comment by JL2210** [comment]
Obviously also changing the comment about not being supported on AArch64.

File: packages/code.rb [hunk 1]
```diff
@@ -3,17 +3,19 @@
...
   when 'i686'
+    version '1.25.1'
     source_url 'https://vscode-update.azurewebsites.net/1.25.1/linux-ia32/stable'
     source_sha256 'af6adc2e2500e50bfebe7ee7b97d661b6e774a590136bf5f89334132a5b292e2'
   else
-    source_url 'https://raw.githubusercontent.com/Microsoft/vscode/1.25.1/README.md'
-    source_sha256 'c1c5e6ec903730a4e116c1cfd83bb29acd227195d97f2ec8944452808232c310'
+    version '1.28.0'
+    source_url 'https://github.com/fsx950223/vscode-arm/releases/download/1.28.0/code.tar.gz'
+    source_sha256 '98260cb89024e2bbd61edb5ed620f29daefc1bcbaa041cca249edc446baebe7c'
```
Comment (uberhacker) [inline packages/code.rb]: We should not be removing the version in this package.  Have you tried to see if code will install via dpkg on any other architecture?  I'm not a big fan of using Debian packages to create binaries if we can successfully compile from source or use the official pre-built binaries from Azure.  Can you please describe the steps you performed to create your pre-built binary for arm?

Comment (JL2210) [reply packages/code.rb]: He's not using `dpkg`. He's using a tarball.

Comment (uberhacker) [reply packages/code.rb]: I'm willing to bet money that he built this Chromebrew package by installing via dpkg.

Comment (uberhacker) [reply packages/code.rb]: Dude, you don't understand what I'm asking here.  He created that tarball somehow.  I bet he installed using dpkg and then added the files to build his Chromebrew package.

Comment (JL2210) [reply packages/code.rb]: So? How is that bad? The only things Debian screws up are in the `control.tar.gz` file. That's not in there. He could've very well built it again, also.

Comment (JL2210) [reply packages/code.rb]: Anyway, @fsx950223, please build version 1.25.1 for arm, see my comments above, and this is good.

**Comment by fsx950223** [comment]
@JL2210 PTML

File: packages/code.rb (hunk 1 above)
Comment (fsx950223) [reply packages/code.rb]: @JL2210 I build the version 1.21.1

Comment (fsx950223) [reply packages/code.rb]: Anyone could have a test.There is something wrong with my chromebook

**Comment by JL2210** [comment]
@fsx950223 We need 1.25.1, not 1.21.1

**Comment by JL2210** [comment]
Also, just remove the part about aarch64, not the entire comment.

File: packages/code.rb (hunk 1 above)
Comment (fsx950223) [reply packages/code.rb]: @uberhacker The package isn't depend on dpkg. I get the binary files from deb and compress it again.

**Comment by fsx950223** [comment]
@JL2210 I get binary files from  https://github.com/stevedesmond-ca/vscode-arm/releases and there hasn't version 1.25.1

**Comment by sajjaadf** [comment]
Yes please do

File: packages/code.rb (hunk 1 above)
Comment (sajjaadf) [reply packages/code.rb]: I have tried, but @fsx950223 when run command "code" I get this error `/usr/local/bin/code: line 35: /usr/local/share/code/bin/../code: Permission denied`

**Comment by sajjaadf** [comment]
I have tried, but @fsx950223 when run command "code" I get this error /usr/local/bin/code: line 35: /usr/local/share/code/bin/../code: Permission denied

**Comment by sajjaadf** [comment]
Actully now it is (code:3475): Gtk-WARNING **: 21:32:50.196: cannot open display: :0

**Comment by fsx950223** [comment]
I will reopen it when the package is available

**Comment by sajjaadf** [comment]
WHY?

**Pull request closed without merging**